vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/model/implementation/Serializer.java

```diff
@@ -43,15 +43,22 @@
  *
  */
 public final class Serializer {
-	public static final String negativeIdentifier = "~";
-	public static final String comma = ",";
-	public static final String dot = ".";
-	public static final String existentialIdentifier = "!";
-	public static final String universalIdentifier = "?";
-	public static final String namedNullIdentifier = "_";
-	public static final String openParenthesis = "(";
-	public static final String closingParenthesis = ")";
-	public static final String ruleSeparator = ":-";
+	public static final String NEGATIVE_IDENTIFIER = "~";
+	public static final String COMMA = ", ";
+	public static final String DOT = ".";
+	public static final String EXISTENTIAL_IDENTIFIER = "!";
+	public static final String UNIVERSAL_IDENTIFIER = "?";
+	public static final String NAMEDNULL_IDENTIFIER = "_";
+	public static final String OPEN_PARENTHESIS = "(";
+	public static final String CLOSING_PARENTHESIS = ")";
+	public static final String RULE_SEPARATOR = " :- ";
+	public static final String AT = "@";
+	public static final String SOURCE = "@source ";
+	public static final String COLON = ": ";
+	public static final String CARET = "^";
+	public static final String LESS_THAN = "<";
+	public static final String MORE_THAN = ">";
+	public static final String ESCAPED_QUOTE = "\"";
```

Comment:
Why is this called `ESCAPED_QUOTE`? The String just contains a single quote character (the escaping happens on the Java syntax level here).